Client/Ecosia/UI/PageAction/PageActionMenuCell.swift

```diff
@@ -0,0 +1,225 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+final class PageActionMenuCell: UITableViewCell {
+    
+    struct UX {
+        
+        /// The cell's identifier
```

Comment:
While I am generally in favour of constructive inline documentation, I'm a bit on the fence with this one. Imho it does not add value to the property name itself. Thus I'd consider those kind of comments redundant and also maintenance overhead. No extra need to remove this but I would like to settle on an agreement where we only add comments where they add substantial information, agree? 🤝

Reply:
💯